package.json

```diff
@@ -25,17 +26,19 @@
         "htmllint": "./node_modules/.bin/htmlhint --config .htmlhintrc.json src",
         "eslint": "./node_modules/.bin/eslint src",
         "tests": "mocha --require babel-register tests/test.js",
-        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js"
+        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js",
+        "eStart": "electron ."
```

Comment:
I could be wrong but isn't the convention to just have `"electron": "electron ."`? That way when you build, you run `npm run electron`.

Reply:
It is I just wanted something shorter I will change it.